source/broadband/webpa_rbus.c

```diff
@@ -124,3 +129,51 @@ rbusError_t clearTraceContext()
 		WalError("Rbus not initialized in clearTraceContext funcion\n");
         }
 }
+
+static void CloudConnOnlineCallbackHandler(
+    rbusHandle_t handle,
+    rbusEvent_t const* event,
+    rbusEventSubscription_t* subscription)
+{
+
+    rbusValue_t newValue = rbusObject_GetValue(event->data, "value");
+    
+    int incoming_value = rbusValue_GetInt32(newValue);
+
+    WalInfo("Received cloud online callback event %s\n", event->name);
+
+    if(incoming_value)
+    {
+	WalPrint("Received cloud connection online event\n");
+	pthread_mutex_lock (get_global_sync_mutex());
+	//Triggering cloud connection online event for retrying sync notification.
+	pthread_cond_signal(get_global_sync_condition());
+	pthread_mutex_unlock(get_global_sync_mutex());
+    }
+    (void)handle;
+}
+
+void SubscribeCloudConnOnlineEvent()
+{
+	int rc = RBUS_ERROR_SUCCESS;
+	WalPrint("================= CloudConnOnlineEventSubscribeHandler ==========\n");	
+    	WalInfo("rbus event subscribe to cloud connection online subscribe callback\n");
+    	if(isRbusInitialized)
+    	{
+    		rc = rbusEvent_Subscribe(rbus_handle, CLOUD_CONN_ONLINE, CloudConnOnlineCallbackHandler, NULL, 0);
+    		if(rc != RBUS_ERROR_SUCCESS)
+		{
+			WalError("consumer: rbusEvent_Subscribe for %s failed: %d\n", CLOUD_CONN_ONLINE, rc);
+			return NULL;
+		}
+		else
+		{
+			WalInfo("consumer: rbusEvent_Subscribe for %s success\n", CLOUD_CONN_ONLINE);
+		}
+	}
+	else 
+	{
+		WalError("Rbus not initialized in CloudConnOnlineEventSubscribeHandler funcion\n");
```

Comment:
Correct the func name in this log

Reply:
Error log should be "Failed to subscribe to cloud_conn_online event as rbus is not initialized"